index.html

```diff
@@ -48,112 +54,156 @@ <h2>I’am a Software Developer</h2>
+                        </p>
+                        <ul>
+                            <li class="html2">html</li>
+                            <li class="bootstrap2">bootstrap</li>
+                            <li class="Ruby2">ruby</li>
+                        </ul>        
+                    </div>
+                    <div class="action-btn"><button class="second-btn" type="button">See Project</button></div>
+                </div>
+                </div>
+                <div class="card2">
+                    <div class="style-grid">    
+                        <div class="profession profession4">
+                            <h4 class="pro">Profesional Art Printing Data</h4>
+                            <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                            </p>
+                            <ul>
+                                <li class="html2">html</li>
+                                <li class="bootstrap2">bootstrap</li>
+                                <li class="Ruby2">ruby</li>
+                            </ul>        
+                        </div>
+                        <div class="action-btn"><button class="second-btn" type="button">See Project</button></div>
+                    </div>
+                    <div class="style-grid">
+                        <div class="profession profession2">
+                            <h4 class="pro">Profesional Art Printing Data</h4>
+                            <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                            </p>
+                            <ul>
+                                <li class="html2">html</li>
+                                <li class="bootstrap2">bootstrap</li>
+                                <li class="Ruby2">ruby</li>
+                            </ul>        
+                        </div>
+                        <div class="action-btn"><button class="second-btn" type="button">See Project</button></div>
+                    </div>
+                    <div class="style-grid">
+                        <div class="profession profession3">
+                            <h4 class="pro">Profesional Art Printing Data</h4>
+                            <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                            </p>
+                            <ul>
+                                <li class="html2">html</li>
+                                <li class="bootstrap2">bootstrap</li>
+                                <li class="Ruby2">ruby</li>
+                            </ul>        
+                        </div>
+                        <div class="action-btn"><button class="second-btn" type="button">See Project</button></div>
```

Comment:
- [ ] [OPTIONAL] i would suggest that you add a `cursor: pointer;` style to all of your button to change the cursor type when users hover on the buttons, that way your buttons will be more elegant and user friendly

Before            |  After
:-------------------------:|:-------------------------:
![G1](https://user-images.githubusercontent.com/89282221/186756877-f7657a5c-d2ee-4421-ac8d-b34d524495eb.gif)  |  ![G2](https://user-images.githubusercontent.com/89282221/186756884-438d4a24-ddd3-40bc-87aa-73ced259cc57.gif)